fms2_io/netcdf_io.F90

```diff
@@ -32,6 +32,7 @@ module netcdf_io_mod
 use mpp_mod
 use fms_io_utils_mod
 use platform_mod
+use mpi, only: MPI_INFO_NULL
```

Comment:
This MPI dependency should be removed.  It can be made public through mpp

Reply:
I will remove this dependency once FMS provides the needed functionality via some other means.